src/assets/styles/custom-scroll.less

```diff
@@ -4,6 +4,8 @@
 
 .side-scroll {
   padding-bottom: 0px;
+  border-top: 1px solid @dark-background;
+  transition: border 0.7s;
```

Comment:
@mic-smith It doesn't really add much, it just adds a bit of consistency -- but it's certainly not something I'd insist on